test/unit/storage-component.spec.ts

```diff
@@ -0,0 +1,202 @@
+import { AuthIdentity } from '@dcl/crypto'
+import { createStorageComponent } from '../../src/ports/storage/component'
+import { IStorageComponent } from '../../src/ports/storage/types'
+import { createTestIdentity, generateRandomIdentityId } from '../utils/test-identity'
+
+describe('when testing storage component', () => {
+  let storage: IStorageComponent
+  let validAuthIdentity: AuthIdentity
+  let identityId: string
+
+  beforeEach(async () => {
+    storage = createStorageComponent()
+
+    identityId = generateRandomIdentityId()
+
+    const testIdentity = await createTestIdentity()
+    validAuthIdentity = testIdentity.authChain
+  })
+
+  describe('when storing identity IDs', () => {
+    describe('and the identity ID is valid', () => {
```

Comment:
As there's no check for the identity to be valid when storing it, WDYT about merging both scopes into one? `when storing an identity` => `should store it in the store`

The `and the identity ID does not exist` might be better to be out of the scope as `when getting an identity that is not stored` => `should return null`